Q6.py

```diff
@@ -0,0 +1,12 @@
+def count_substring(string, substring):
+
+    #Using list comprehension and slicing , i have count the occcurence of substring in string. and then i have append 1 in list,
+    #after the i have sum all using sum function
+    count = sum([1 for _ in range(0,len(string)) if string[_:len(substring)+_] == substring])
```

Comment:
You are using `_` in the code. So please give it a proper name, because `_` can be used when you don't have to use that.